templates/general/instructions-for-use.md

```diff
@@ -114,6 +114,8 @@ This symbol indicates supplementary information.
 
 ## Safety and Maintenance
 
+The software's lifetime is established to be [for example: three years]. This is what is expected to be the maximum time until the implementation of a significant change, by which the manufacturer is able to react to the relevant changes to the software device environment, such as SOUP changes, cybersecurity innovations, or the evolving technological or medical state of the art.
```

Comment:
I see this questions causing problems at various clients. I agree with how this is phrased to indicate that most software doesn't have infinite shelf-life, since the ecosystem it tends to rely on changes or vulnerabilities are found requiring updates to the software

Here's a question a client recently got from BSI:
> The reviewer is aligned with the defined clinical lifetime of the device. Per MDR, Annex XIV, Part B, the PMCF activities should collect and evaluate data with, "the aim of confirming the safety and performance throughout the expected lifetime of the device." Would you please point to or provide evidence of what specific data collection will be done, including collection of data to support the 3 year intended clinical lifetime in use?

So I think you want to make clear that this lifetime is separate from how long you intend to monitor the device as part of PMCF.

Something along the lines of:
> We are committed to conducting PMCF activities for the duration of the device's presence on the market, irrespective of the initial three-year period. These activities will be adjusted and scaled based on real-world data, technological advancements, and regulatory changes.

> Our plan includes annual software updates to enhance functionality, address emerging risks, and maintain compliance with the latest safety standards. These updates are part of our proactive approach to ensure that the device remains safe and effective throughout its market life.

> Beyond the specific PMCF activities planned for the initial years, our risk management strategy involves continuous surveillance and iterative updates to the risk management file. This ongoing process will ensure that we remain responsive to new information and maintain compliance with regulatory requirements.

Though of course you wouldn't put that in your IFU, but if you make statements about this, be prepared for such pedantic questions. I've seen similar questions on OpenRegulatory's Slack as well

Reply:
Thanks a lot for the pointer, Ivo! I understand that it might make sense to clarify a difference between estimated product lifetime and planned PMCF surveillance periods. I've added a sentence along your lines to the risk plan, where the device time also plays a role, because that is more of an internal information and the risk plan might be less public than the IFU:
https://github.com/openregulatory/templates/pull/49/commits/9f95260d7300e320ea1799e5463bc04507994cb1